engine/schema/src/main/resources/META-INF/db/schema-41200to41300.sql

```diff
@@ -27,6 +27,9 @@ INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid, hypervisor_type, hype
 
 -- Add VMware 6.7 hypervisor capabilities
 INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid,hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) VALUES (UUID(), 'VMware', '6.7', '1024', '0', '59', '64', '1', '1');
+INSERT IGNORE INTO `cloud`.`hypervisor_capabilities`(uuid,hypervisor_type, hypervisor_version, max_guests_limit, security_group_enabled, max_data_volumes_limit, max_hosts_per_cluster, storage_motion_supported, vm_snapshot_enabled) VALUES (UUID(), 'VMware', '6.7.1', '1024', '0', '59', '64', '1', '1');
```

Comment:
@PaulAngus have you tested if adding the minor version makes any difference? That is will a 6.7.1/6.7.2/6.7.3 be still recognised by CloudStack as 6.7?

Reply:
6.7.1, 6.7.2, 6.7.3 are how updates 1,2 and 3 report their hypervisor version to CloudStack.  
Currently, CloudStack does not recognise 6.7 update 1 or update 2 as it is looking for "6.7".